airbyte-webapp/src/views/Connector/ConnectorCard/useAnalyticsTrackFunctions.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback } from "react";
+
+import { Action, Namespace } from "core/analytics";
+import { Connector, ConnectorDefinition } from "core/domain/connector";
+import { useAnalyticsService } from "hooks/services/Analytics";
+
+export const useAnalyticsTrackFunctions = (formType: "source" | "destination") => {
```

Comment:
`connectorType` instead of `formType`?

Reply:
oh yes... I used the "name context" from the form... Will fix it!